cmd/clef/main.go

```diff
@@ -583,7 +583,7 @@ func accountImport(c *cli.Context) error {
 	}
 	if first != second {
 		//lint:ignore ST1005 This is a message for the user
-		return errors.New("Passwords do not match")
+		return errors.New("passwords do not match")
```

Comment:
Could you please revert this change? I guess it's some message being printed for users. I would prefer to keep it as now.

Reply:
Done!

Reply:
Okay Done!